test/e2e/lib/browser-manager.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import playwright from 'playwright';
+
+const browserStartTimeoutMS = 2000;
+
+/**
+ * Returns the target screen size for tests to run against.
+ *
+ * By default, this function will return 'desktop' as the target.
+ * To specify another screen size, set the BROWSERSIZE environment variable.
+ *
+ * @returns {string} String representation of the target screen size.
```

Comment:
I'm curious if Typescript has a place for writing tests. Idk if it's possible, but it might be worth checking out as we do this migration 🤔

Reply:
It could be a good idea. Playwright has native support for TypeScript, though the limiting factor may be me - I do not know TS :P

Reply:
Gotcha! That's good to know. If it wasn't too high impact, I almost feel like it would be worth it to create our initial framework and examples using TypeScript -- but that assumes that we _want_ people to write e2e tests in TS long-term. Do we?

cc @sarayourfriend @jsnajdr

Reply:
Even if e2e tests aren't written in TypeScript long-term (I often find writing tests in TypeScript annoying) it's _always_ helpful for test utilities to be strongly and correctly typed (there are some errors in the types applied here). For example, `jest` exports very helpful type definitions that make exploring the library fun and easy for new comers as long as they're using an editor that supports intellisense.

Reply:
> Even if e2e tests aren't written in TypeScript long-term (I often find writing tests in TypeScript annoying) it's _always_ helpful for test utilities to be strongly and correctly typed (there are some errors in the types applied here).

I am coming new to ES6 (previously a Python developer) so I'm trying to wrap my mind around JS types in general. With that said, I would be happy to refactor `browser-manager.js` to be in TypeScript.